### DISCUSSION TITLE
Should we skip `<notype>` in inlay hints?

### DISCUSSION THREAD

**Discussion by ckipp01** [scalameta/metals#6172] [General]
I noticed that sometimes when I'm navigating around the inlay hints of the inferred types looks like this:

<img width="603" alt="Screenshot 2024-02-28 at 10 05 22" src="https://github.com/scalameta/metals/assets/13974112/098262cc-6383-4759-8eaf-b8158acc725e">

Often it just takes a refresh or to change something for it to correct itself, but it makes me wonder if we should maybe just filter out `<notype>` on the server side and not send these? Wdyt?

**Comment by tgodzik** [comment]
On one hand it makes sense, but on the other I actually found it useful to see when the code fixes itself enough for the presentation compiler to work :sweat_smile: It's a useful indication that something is off.
